sdk/highlight-py/highlight_io/sdk.py

```diff
@@ -436,7 +437,8 @@ def log_hook(self, span: Span, record: logging.LogRecord, formatted: str = ""):
             attributes["highlight.trace_id"] = request_id
             attributes["highlight.session_id"] = session_id
             if isinstance(record.args, dict):
-                attributes.update(record.args)
+                addedAttributes = flatten_dict(record.args, sep=".")
+                attributes.update(addedAttributes)
```

Comment:
would it be easy to update the [sdk unit test](https://github.com/highlight/highlight/blob/946928c433150c0272f8d3695b7874475f0d2204/sdk/highlight-py/tests/test_sdk.py#L77-L85) as well to flex this logic?

Reply:
that would also ideally improve the coverage of the new `sdk/highlight-py/highlight_io/utils/dict.py` file